runner/src/server/plugins/engine/pageControllers/SummaryPageController.ts

```diff
@@ -129,6 +148,26 @@ export class SummaryPageController extends PageController {
         state,
         request
       );
+
+      const applicationId = state.metadata?.application_id ?? "";
+      if (applicationId) {
+        for (const detail of summaryViewModel.details) {
+          const comp = detail.items.find(
+            (c) => c.type === "ClientSideFileUploadField"
+          );
+          if (comp) {
+            const folderPath = `${comp.pageId}/${comp.name}`;
+            const files = await uploadService.listFilesInBucketFolder(
+              `${applicationId}${folderPath}`
+            );
+            comp.value = {
+              folderPath,
+              files,
+            };
```

Comment:
This is used on the summary page, where we'll eventually list all of the files uploaded as well as a link allowing the user to download and preview them.